apps/website/src/components/table-issues/filters/filter-with-checkboxes.tsx

```diff
@@ -0,0 +1,139 @@
+import { cloneElement, useState } from 'react'
+
+import { Avatar, Button, Input, Text } from '@status-im/components'
+import { DropdownMenu } from '@status-im/components/src/dropdown-menu'
+import { DropdownIcon, SearchIcon } from '@status-im/icons'
+
+import { useCurrentBreakpoint } from '@/hooks/use-current-breakpoint'
+
+import { ColorCircle } from './components/color-circle'
+
+import type { ColorTokens } from '@tamagui/core'
+
+type Data = {
+  id: number
+  name: string
+  avatar?: string | React.ReactElement
+  color?: ColorTokens | `#${string}`
+}
+
+type Props = {
+  data: Data[]
+  label: string
+  noPadding?: boolean
```

Comment:
with gap property now supported on grid and flex layouts, individual components don't ever have to deal with setting padding / margins themselves